### ISSUE TITLE
Include deploy example

### ISSUE THREAD

**Issue by AndreJoaquim** [chromaui/intro-storybook-ember-template#1]
Hey 👋🏻 

I've been trying to deploy an Ember.js' Storybook to Chromatic but I get stuck every time in "Verifying your Storybook". After speaking to Varun from Chromatic, he told me the issue is related to console errors.

First, I've built Storybook

```bash
$ build-storybook
```

and visited Storybook at

```bash
$ npx serve storybook-static
```

If there are any errors in the console, it prevents the deploy from happening.

I've reached this repository hoping it had a guide on how-to deploy to Chromatic or, at least, some usual mistakes 😄  Therefore, I think it'd be very useful if this repository contained a real deploy to Chromatic and a guide on how-to do it

Thanks!

**Comment by jonniebigodes** [comment]
Boas @AndreJoaquim!

I've been monitoring the conversation that you had with Varun regarding the issues you've faced while deploying the project to Chromatic. A couple of things regarding your issue:
- This template is aimed at getting you started and avoid having to at the lack of a better word jump through hoops to get started with the tutorial. Including the example for deployment is beyond the scope of this repository. 
- If you follow the tutorial and reach this [stage](https://storybook.js.org/tutorials/intro-to-storybook/ember/en/deploy/) the deployment step works. I'll leave you a small [reproduction](https://github.com/jonniebigodes/intro-storybook-ember-with-template)!

Key takeaways from this:
- Avoid using the `serve` package as a test case to check your production build of Storybook, we have it documented [here](https://storybook.js.org/docs/react/workflows/faq#i-see-a-no-preview-error-with-a-storybook-production-build). As with a recent release generates an issue with Storybook based on its rewrite rules and lead you to a misleading once you preview your production build.
- One thing that came to mind while checking your issue and the conversation is the following, you said that you ran `build-storybook` correct? But here's the thing, do you have a script or something of that nature that will build the Ember app a priori? The reason I'm asking this is based on the fact who Ember and Storybook work together. If you check this repo's `package.json` or the example I've mentioned above you'll see that I have a `prebuild-storybook` script in place. The same applies to development mode. If you look carefully you'll see that there's a `storybook-dev` script as well. Starting to see the pattern? To sum it up Storybook for Ember requires a build to be generated first, then it will do its thing. Based on testing before and after releasing the Ember version of the tutorial with that script ( the `prebuild` one) i was able to successfully deploy Storybook into Chromatic.
- One other thing that came to mind was the fact that you were building your Storybook into the `dist` folder. My recommendation is that you should avoid that. You might ask why. It's simple, Ember will build the app into that directory and Storybook will also, but it could lead to some unforeseen re-writes of either files or folders or some issues with the build itself.


Let me know if you have any further questions about the tutorial or this repo's code and I'll gladly answer them to the best of my knowledge.

Stay safe

**Comment by AndreJoaquim** [comment]
Boas @jonniebigodes!

In the meantime, I was (somehow) able to deploy my application to Chromatic, and I've went through some of the issues you've stated here 🥳 

I don't actually know what got it to work, but basically I build the Ember application with `--environment=development` and then I build the Storybook into `storybook-static` directory, using the previously mentioned Ember build. After that, running Chromatic works just fine.

Nevertheless, I think your questions (and answers!) will be helpful to whoever comes to this repository looking for answers. (I took a large time finding out about `serve` 😂  specially because Varun told me to use it)

Thanks for your time!
